src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -2126,6 +2126,13 @@ public struct ITaskBarList2VTable
             public IntPtr SetActiveAlt;
             public IntPtr MarkFullscreenWindow;
         }
+
+        [DllImport("uxtheme.dll", EntryPoint = "#95", CharSet = CharSet.Unicode)]
+        internal static extern uint GetImmersiveColorFromColorSetEx(uint dwImmersiveColorSet, uint dwImmersiveColorType, bool bIgnoreHighContrast, uint dwHighContrastCacheMode);
```

Comment:
@PieroCastillo 
It's indeed undocumented API, so I would make two things here:
1. Call it only if we are on Win10+ (although it **should** work on earlier versions, it should be checked on all of them otherwise).
2. Add related boolean property to Win32PlatformOptions, so it will be possible to disable this feature for windows. Probably even disable by default right now, since it's kind of visual breaking change. We will be able to enable it by default again later.

Reply:
Given the whole "immersive" thing was added in Windows 8, I highly doubt this API even exists on Windows 7.

Reply:
@maxkatz6  good idea, but I don't know exactly how to work that...

Reply:
We have static ```Win32Platform.WindowsVersion```.
And following code will check for Win10+ version:
```if (Win32Platform.WindowsVersion.Major >= 10)```

Reply:
And about Win32PlatformOptions.

You need to add new bool property to this class https://github.com/AvaloniaUI/Avalonia/blob/master/src/Windows/Avalonia.Win32/Win32Platform.cs#L40

And then you will be able to use it with 
```var options = AvaloniaLocator.Current.GetService<Win32PlatformOptions>();```

Reply:
It's done